pkgs/os-specific/linux/kernel/manual-config.nix

```diff
@@ -128,10 +137,10 @@ let
       ++ optionals withRust [ rustc rust-bindgen ]
       ;
 
-    in (optionalAttrs isModular { outputs = [ "out" "dev" ]; }) // {
+    in (optionalAttrs isModular { outputs = [ "out" "dev" ] ++ (lib.attrValues targetsToOutput); }) // {
```

Comment:
We would end up wanting to have outputs even if the kernel wasn't built with module support.